zipline/utils/formatting.py

```diff
@@ -0,0 +1,45 @@
+def s(word, seq, suffix='s'):
```

Comment:
`zipline.pipeline.domain.bulleted_list` would probably be another good candidate for this file.

Reply:
Do you think I should go move that now?

Reply:
/shrug. I don't have a strong preference.